lib/heimdall_tools/nessus_mapper.rb

```diff
@@ -96,14 +117,15 @@ def nist_tag(pluginfamily, pluginid)
     end
 
     def impact(severity)
+      # Map CAT levels and Plugin severity to HDF impact levels
       case severity
       when "0"
         IMPACT_MAPPING[:Info]
-      when "1"
+      when "1","III"
         IMPACT_MAPPING[:Low]
-      when "2"
+      when "2","II"
         IMPACT_MAPPING[:Medium]
-      when "3"
+      when "3","I"
         IMPACT_MAPPING[:High]
       when "4"
         IMPACT_MAPPING[:Critical]
```

Comment:
CAT 4 or IIII is 'none' or 'info'

Reply:
Nessus report has no concept for CAT 4;
`info` level is placed based on the `plugin severity` 0